src/gossip_agent/mod.rs

```diff
@@ -82,11 +84,13 @@ impl GossipAgent {
             query_network_subgraph(NETWORK_SUBGRAPH.to_string(), indexer_address.clone())
                 .await?
                 .indexer_allocations();
-        let subtopics = indexer_allocations
-            .iter()
-            .map(|s| &**s)
-            .collect::<Vec<&str>>(); // subtopics allocations
-                                     // .collect::<Vec<&str>>()[1..2].to_vec();
+
+        let subtopics = subtopics.unwrap_or_else(|| {
```

Comment:
Since filter for content topic doesn't work yet, operator will save nonces for content topic (radio-name-sub-topic) from other radio channels as well. Attestations should work fine still but the message cache is unnecessary. I think a nice improvement is for operator to ignore messages of other use cases, but can wait til next issue if it isn't a quick change:)

Reply:
What do you mean "message cache is unnecessary"? 👀 Also by "ignore messages of other use cases" you mean a check before saving them to the message store right? That should be pretty easy

Reply:
By "message cache is unnecessary", I meant that when the radio operator should not stores the message for attestation later when the message is for a different radio instance. For example 'ping-pong' radio shouldn't find a use for message with content topic 'poi-crosschecker'.

I think the check should happen before checking message validity, where the gossip operator first check that message is for the intended radio, and then validate.

Reply:
this should be fixed now